<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo Gallery</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_0_front">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_images">
           <attribute name="title">
            <string>Images</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page_1">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_picture">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QListWidget" name="listImages">
                  <property name="movement">
                   <enum>QListView::Static</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Show All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbAddtoalbum">
                <property name="text">
                 <string>Add to album</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbRotate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Rotate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbBack">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbSlidesShow">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Slides show</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbNext">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbExit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbEdit">
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>60</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_albums">
           <attribute name="title">
            <string>Albums</string>
           </attribute>
           <widget class="QPushButton" name="pushButton">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>111</width>
              <height>41</height>
             </rect>
            </property>
            <property name="text">
             <string>Add new album</string>
            </property>
           </widget>
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>-30</y>
              <width>899</width>
              <height>427</height>
             </rect>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="tab_images_2">
             <attribute name="title">
              <string>Images</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QStackedWidget" name="stackedWidget_3">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="page_3">
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_picture_2">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_4">
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QListWidget" name="listImages_2">
                    <property name="movement">
                     <enum>QListView::Static</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_albums_2">
             <attribute name="title">
              <string>Albums</string>
             </attribute>
             <widget class="QPushButton" name="pushButton_2">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>40</y>
                <width>111</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>Add new album</string>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>59</width>
                <height>15</height>
               </rect>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="albumName">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>20</y>
                <width>141</width>
                <height>21</height>
               </rect>
              </property>
             </widget>
             <widget class="QListWidget" name="AlbumListWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>90</y>
                <width>831</width>
                <height>231</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>51</y>
                <width>71</width>
                <height>20</height>
               </rect>
              </property>
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="DescriptionLine">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>50</y>
                <width>141</width>
                <height>21</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_4">
              <property name="geometry">
               <rect>
                <x>700</x>
                <y>50</y>
                <width>101</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Remove Album</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="removeAlbum">
              <property name="geometry">
               <rect>
                <x>570</x>
                <y>50</y>
                <width>113</width>
                <height>23</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>520</x>
                <y>40</y>
                <width>61</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_1_front">
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pbFullScreenExit">
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelSlide">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuFilters">
     <property name="title">
      <string>Filters</string>
     </property>
     <addaction name="actionBy_Date"/>
     <addaction name="separator"/>
     <addaction name="actionBy_Na_e"/>
     <addaction name="separator"/>
     <addaction name="actionBy_Type"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menuFilters"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAlbums">
   <property name="text">
    <string>Albums</string>
   </property>
  </action>
  <action name="actionBy_Date">
   <property name="text">
    <string>By Date</string>
   </property>
  </action>
  <action name="actionBy_Na_e">
   <property name="text">
    <string>By Name</string>
   </property>
  </action>
  <action name="actionBy_Type">
   <property name="text">
    <string>By Type</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
